index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Baloo+Bhaijaan+2&display=swap" rel="stylesheet">
+<style>
+h1 , h2 , h3 {text-align: center;}
+*{
+  box-sizing: border-box;
+}
+#column {
+  float: left;
+  width: 33%;
+  padding: 5px;
+}
+body{
+  font-family: 'Baloo Bhaijaan 2', cursive;
+}
+
+.row::after {
+  content: "";
+  clear: both;
+  display: table;
+}
+p{
+  font-size: 23px;
+}
+
+    </style>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <script src="https://kit.fontawesome.com/c8fd53be77.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <h3 style="color: blueviolet;">Our News </h3>
+<h1>Our Latest News</h1>
+<div class="row">
+    <div id="column">
+      <img src="https://landing.zytheme.com/ladidapp/assets/images/blog/1.jpg" alt="Slow Internet Connection" style="width: 95%;">
+      <br>  <i class="fas fa-user-circle" style="color: darkorchid;"> Zytheme </i>
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+
+
+      <i class="fas fa-calendar-week" style="color: darkorchid;"> 20 August 2020</i> 
+     <p> <b> Design your apps in your own web Strateaies </b> </p>
+    </div>
+    <div id="column">
+      <img src="https://landing.zytheme.com/ladidapp/assets/images/blog/2.jpg" alt="Slow Internet Connection" style="width: 95%;">
+      <br> <i class="fas fa-user-circle" style="color: darkorchid;"> Zytheme </i>
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```

Comment:
Instead of writing so many &nbsp, you can add or format the spacing using div and CSS align property.